template.php

```diff
@@ -425,3 +425,18 @@ function _campaignion_foundation_exclude_block_from_contextual_links($module) {
   ];
   return in_array($module, $exclude);
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter() for webform_client_form().
+ *
+ * Provide the form step information in Drupal.settings for the behavior
+ * JavaScripts to pick up.
+ */
+function campaignion_foundation_form_webform_client_form_alter(&$form, &$form_state, $form_id) {
```

Comment:
Could you please move the function after `campaignion_foundation_form_alter` to group all the form alters together?

Reply:
A bit more up!

Reply:
Should now be correct.

Reply:
I’d still move it farther up:

hook_form_alter()
hook_form_FORM_ID_alter()
hook_payment_forms_payment_form_alter()
hook_webform_component_render_alter()

Makes more sense to me than

hook_form_alter()
hook_payment_forms_payment_form_alter()
hook_webform_component_render_alter()
hook_form_FORM_ID_alter()

Reply:
Changed.